pyproject.toml

```diff
@@ -80,6 +80,10 @@ trino = ["trino"]
 clickhouse = ["clickhouse-driver"]
 vertica = ["vertica-python"]
 duckdb = ["duckdb"]
+all-dbs = [
```

Comment:
Can you comment out this line? This makes it clear why this isn't part of the `all-dbs` extra list. 

```poetry
mssql = ["pyodbc"]
```